.github/workflows/javaapp.yml

```diff
@@ -0,0 +1,31 @@
+name: JUnit Tests
+
+env:
+  API_HOST: ${{ vars.API_HOST }}
+  API_KEY: ${{ secrets.API_KEY }}
+
+on:
+  push:
+    branches: [ master, openapi-bindings ]
+  pull_request:
+    branches: [ master, openapi-bindings ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up JDK 1.7
+        uses: actions/setup-java@v1
+        with:
+          java-version: '7'
```

Comment:
i think we can use a newer version of java. at the very least java 8 but i think even java 11 would be fine